core/frontend/src/utils/pattern_validators.ts

```diff
@@ -0,0 +1,40 @@
+export function validate_port(port: number, public_range = true): boolean {
+  if (typeof port !== 'number') {
```

Comment:
How is the possible with typescript ?

Reply:
Good catch. This was from when I was prototyping. Will remove.